src/agoradatatools/etl/transform/biomarkers.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module contains the transformation logic for the biomarkers dataset.
+This is for the Model AD project.
+"""
+
+import pandas as pd
+
+
+def transform_biomarkers(datasets: dict) -> list:
```

Comment:
nit, are there more specific typing we can add for this aside from a generic `dict` or `list`?

Also - The docstring:
```
    Returns:
        dict: a dictionary of biomarkers data modeled after intended final JSON structure
```

Does not match the return type of the function of `list`. 


An example of how to add more specific typing (Might not be correct based on my comment above):

```
from typing import Dict, List

def transform_biomarkers(datasets: Dict[str, str]) -> List[str]:
```

Reply:
Oops! Thank you for catching this! I updated the typing hints and the docstring for the function :)

Reply:
@BryanFauble @beatrizsaldana If I recall from previous experience, this isn't possible to do at this time. Adding type hints like `list[str]` isn't supported by Python 3.8 so our CI fails on it. Unfortunately we have to leave it generic.

Reply:
Ohh, I have been using Python 3.9. I'll revert the typing hints back to generic dict, list, etc. Thank you! :)

Reply:
You're fine to keep using 3.9, I think it's what gets installed by default when installing agora data tools, but our CI runs the test suite on 3(?) different Python versions, including 3.8, so sometimes we run across stuff that fails in 3.8 but not 3.9. I've run into this type hinting issue before haha.

Reply:
Thank you! You probably saved me a lot of future failed debugging attempts :)

Reply:
@jaclynbeck-sage @beatrizsaldana 
It is supported in Python 3.8 but not the way you are suggesting: `Adding type hints like list[str] isn't supported by Python 3.8 so our CI fails on it`

The following does work in Python 3.8:
```
from typing import Dict, List

def transform_biomarkers(datasets: Dict[str, str]) -> List[str]:
```

Reply:
Ohh interesting. Learned something new! Thanks Bryan.

Reply:
I'll use the typing library since I do really like to be specific with type hinting.